scripts/vmpublish.rb

```diff
@@ -0,0 +1,80 @@
+require 'logger'
```

Comment:
Can we name this `publish.rb`?  We really need to rename vmbuild.rb to build.rb since that file adds the `Build` module.